DVR/HTTPURLResponse.swift

```diff
@@ -1,18 +1,18 @@
 import Foundation
 
-// There isn't a mutable NSHTTPURLResponse, so we have to make our own.
-class URLHTTPResponse: NSHTTPURLResponse {
+// There isn't a mutable HTTPURLResponse, so we have to make our own.
+class HTTPURLResponse: Foundation.HTTPURLResponse {
```

Comment:
I had a similar concern but since this isn't a part of the public API, I think we can address it later and not block merging this.